packages/flutter_tools/lib/src/commands/build_apk.dart

```diff
@@ -29,6 +30,11 @@ const String _kDefaultResourcesPath = 'android/res';
 const String _kFlutterManifestPath = 'flutter.yaml';
 const String _kPackagesStatusPath = '.packages';
 
+// Files generated by the ahead-of-time snapshot builder.
+const List<String> _kAotSnapshotFiles = const <String>[
```

Comment:
Should these definitions live in `build_aot.dart` then? Or in some aot specific file, `lib/src/aot.dart`?